devtools-ci-index.yaml

```diff
@@ -898,6 +898,12 @@
             timeout: '30m'
             svc_name: keycloak-server
         - '{ci_project}-{git_repo}':
+            git_organization: almighty
+            git_repo: keycloak
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '20m'
```

Comment:
hmm! I don't think so, we got errors of parts missing when only building the server. Is there anything new in KC master that removes those dependencies ?